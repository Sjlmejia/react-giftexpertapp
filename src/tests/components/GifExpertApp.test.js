import React from 'react';
import {shallow} from 'enzyme'
import GifExpertApp from '../../GifExpertApp'
describe('pruebas para GiftExperApp', () => {
  test('debe crearse', () => {
    const wrapper = shallow(<GifExpertApp/>);
    expect(wrapper).toMatchSnapshot();
  });

  test('debe mostrarse una lista de elementos', () => {
    const categories =['dragon'];
    const wrapper = shallow(<GifExpertApp defaultCategories={categories}/>);
    expect(wrapper.find('GifGrid').length).toBe(categories.length);
  });
});