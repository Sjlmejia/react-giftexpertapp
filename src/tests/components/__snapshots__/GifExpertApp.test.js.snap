// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pruebas para GiftExperApp debe crearse 1`] = `
<Fragment>
  <h2>
    GifexpertApp
  </h2>
  <AddCategory
    setCategories={[Function]}
  />
  <hr />
  <ol />
</Fragment>
`;
